pallets/rmrk-market/src/lib.rs

```diff
@@ -424,7 +428,7 @@ pub mod pallet {
 					}
 
 					<T as pallet::Config>::Currency::unreserve(&offer.maker, offer.amount);
-					Self::do_buy(offer.maker, collection_id, nft_id, true)?;
+					Self::do_buy(offer.maker, collection_id, nft_id, None,true)?;
```

Comment:
`cargo fmt` please

Reply:
> Didn't understand why checking the price can prevent front running. Btw which kind of front running do you refer to?

This was based on a video tutorial I saw on the Substrate Kitties pallet. It was mentioned that if the buy doesn't include the price, a `buyer` could buy the NFT, but the current `owner` could have updated the price before the `buy` is executed. So the `buyer` could end up buying at a higher listed price if the Storage is updated before the `do_buy` helper function reads the Storage value of the listed price. So if we pass in the price then the buyer will pay what is expected opposed to the price in Storage at the time `do_buy` is executed. Am I thinking about this in the right way or is this not the case in this scenario?

Reply:
Got it. It makes sense

Reply:
New build will have corrected formatting errors with `cargo fmt`